mentorship/gsoc/2022/projects/new-docs-site-engine/README.md

```diff
@@ -88,7 +87,7 @@ Issue link: [https://github.com/keptn-sandbox/new-keptn-docs-engine/issues/2](ht
 
 ---
 
-- [x] Adding UI and UX changes, moving the content and search functionality to the new documentation site engine.
+- [ ] Adding UI and UX changes, moving the content and search functionality to the new documentation site engine.
 
   - [x] Implementing UI and UX changes in the documentation engine like creating components, templates, standalone pages, etc.
```

Comment:
So you are saying that these are actually done?  I'm not sure I understand how they could be verified without the UI/UX design created and implemented and with virtually none of the core documents transferred to the new repo but perhaps it is possible.